src/Hl7.Fhir.Specification/Schema/Binding.cs

```diff
@@ -149,14 +149,14 @@ internal async Task<OperationOutcome> ValidateCode(ITypedElement source, Element
 
 
                 var outcome = (await svc.ValueSetValidateCode(parameters).ConfigureAwait(false)).ToOperationOutcome();
-                foreach (var issue in outcome.Issue) issue.Location = new string[] { location };
+                foreach (var issue in outcome.Issue) issue.Expression = new string[] { location };
```

Comment:
In common, you have set both for backwards compat - here we do not do that, so older code that still looks to Location will fail now, right?